node/src/lib.rs

```diff
@@ -1,5 +1,12 @@
 // Copyright 2021 ChainSafe Systems
 // SPDX-License-Identifier: LGPL-3.0-only
 
+#[cfg(not(any(feature = "kusama", feature = "polkadot")))]
+pub use pint_runtime_dev as pint_runtime;
+#[cfg(feature = "kusama")]
+pub use pint_runtime_kusama as pint_runtime;
+#[cfg(feature = "polkadot")]
+pub use pint_runtime_polkadot as pint_runtime;
```

Comment:
`kusama` and `polkadot` are derived from `dev`, use `dev` runtime by default.

Reply:
I guess that's reasonable for now, but as a next step we should create separate services for each rt